tools/topology/CMakeLists.txt

```diff
@@ -119,6 +119,7 @@ set(TPLGS
 	"sof-imx8-wm8960-mixer\;sof-imx8-wm8960-mixer"
```

Comment:
Mostly looking good. We need to adapt the commit messages to reflect the coding style of the project.

First the commit subject should use some tags relevant for the subsystem being updated. E.g `topology`. Also, use imperative mood in the commit subject. E.g: `Create` instead of `Created`

So, the commit message should look like this:

```
topology: Create a KWD topology for i.MX8MP platform

Adapted the APL KWD topology to iMX8MP by switching to
SCHEDULE_TIME_DOMAIN_DMA on capture pipeline and
creating a no LP capture file for KFBM component.
```